src/core/03_aks.tf

```diff
@@ -29,7 +29,7 @@ module "k8s_snet" {
 }
 
 module "aks" {
-  source = "git::https://github.com/pagopa/azurerm.git//kubernetes_cluster?ref=2.3.1"
+  source = "git::https://github.com/pagopa/azurerm.git//kubernetes_cluster?ref=kv-secrets-autorotation"
```

Comment:
To change after merge of https://github.com/pagopa/azurerm/pull/196

Reply:
```suggestion
  source = "git::https://github.com/pagopa/azurerm.git//kubernetes_cluster?ref=v2.4.0"
```